battle-engine.js

```diff
@@ -1256,6 +1256,23 @@ BattlePokemon = (function () {
 		if (this.battle.gen >= 5) return ['Normal'];
 		return ['???'];
 	};
+	BattlePokemon.prototype.isGrounded = function () {
+		if (!this.hasType('Flying') && !this.hasAbility('levitate') && !this.hasItem('airballoon') && !this.volatiles['magnetrise'] && !this.volatiles['telekinesis']) {
```

Comment:
Except for Flying-types under Inverse Battle, to my knowledge. They're still ungrounded, even though they take 2x from Ground.

This was originally going to be just a PR with Inverse Battle regression tests, and then it failed for this very specific reason.